### PR TITLE
7.x hooked access

### PR DESCRIPTION
**Pull request by adam-vessey**
Ontime #1733.
https://jira.duraspace.org/browse/ISLANDORA-885

### CODE REVIEW

**Comment by adam-vessey** [comment]
Test passing will depend on https://github.com/Islandora/islandora/pull/352.

File: tests/hooked_access.test [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+  }
+
+  /**
+   * Test all XACML operations on object.
+   */
+  public function testAllXacmlAccess() {
+    $user_allow = $this->user_allow;
+    $user_deny = $this->user_deny;
+
+    variable_set('islandora_strict_user_access_enforcement', FALSE);
```
Comment (jonathangreen) [inline tests/hooked_access.test]: Wanna kill this beast?

Comment (adam-vessey) [inline tests/hooked_access.test]: Actually... The entire implementation of this one is useless without this being able to be false... Can just strip it all out...

**Comment by adam-vessey** [comment]
Leaving the Travis stuff out for now, as there are some deeper issues to deal with: https://travis-ci.org/adam-vessey/islandora_scholar-1/jobs/8287761

**Comment by adam-vessey** [comment]
Scratch that, I got Travis passing. Huzzah!

**Pull request merged**